chat-widget/src/components/livechatwidget/common/overridePropsOnMockIfApplicable.ts

```diff
@@ -22,7 +22,7 @@ const overridePropsOnMockIfApplicable = (props: ILiveChatWidgetProps) => {
         props.webChatContainerProps = {
             ...props.webChatContainerProps,
             webChatProps: {
-                disabled: true,
+                disabled: props?.mock?.mockMessages && props?.mock?.mockMessages.length > 0 ? false : true,

```

Comment:
Suggestion
`props?.mock?.mockMessages && props?.mock?.mockMessages.length > 0 ? false : true,`
to
`!(props?.mock?.mockMessages?.length),`